src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -49,6 +50,30 @@ import PrefilledNameInput from '../commons/prefilled-name-input';
 
 const STRING_LIST = 'STRING_LIST';
 
+function customizeCurrentParameters(params) {
+    return params.reduce((obj, parameter) => {
+        // we check if the parameter is for extension, if it is, we select all possible values by default.
```

Comment:
we check if the parameter is for extensions. If so, we select all possible values by default.